README.md

```diff
@@ -116,15 +116,16 @@ Usage documentation
 The extention is provided as-is, but here is some basic documentation of how
 to use it.
 
+Use `-XLinearTypes` to activate the extension. This adds the following syntax:
 
-## Activating the extension
-The extension is _always on_ for the time being, so no need to set any
-flags or pragmas. In the future it will be an ordinary extension.
+* `a ->. b` for linear functions
+* `a ⊸ b` for linear functions, if `-XUnicodeSyntax` is on
+* `a -->.(p) b` for functions of multiplicity `p :: Multiplicity`. This type
+  is defined in `GHC.Types`.
```

Comment:
Add that this last be is highly experimental and expected to break in many cases.